docs/2.9.0/docs/canton/usermanual/kms/external_key_storage/participant_kms_migration/participant_kms_migration.rst

```diff
@@ -5,27 +5,24 @@
 
 .. _participant_kms_migration:
 
-Participant Node Migration
---------------------------
+Participant Node Migration to KMS
+---------------------------------
 
 To migrate an existing participant node connected to a sync domain with a non-KMS-compatible provider
 and start using KMS external keys, you need to follow one of the next guides.
-The general idea is to replicate the old node into a new one that uses a KMS provider and connects to
+The general idea is to replicate the old node into a new one that uses a KMS crypto provider and connects to
 a KMS-compatible sync domain (e.g. :ref:`running JCE with KMS-supported encryption and
 signing keys <external_key_storage_interoperability>`).
 
-.. note::
-
-    This process applies to any type of Participant node migration, not just KMS-related ones.
-
 We offer two different ways of doing this migration: (a): :ref:`migrate to a new participant that uses the same namespace
 identity <participant_kms_migration_same_namespace>` and (b) :ref:`migrate to a new participant with an entire new identity
-and namespace <participant_kms_migration_new_namespace>`. The first option,
-does not require us to change the party ids for the active contracts in the old participant, but both
-old and new domain have to use the same protocol version. Furthermore, the original root namespace key must
-be kept safe to allow the future migration to a new protocol version. The second option re-writes the contracts
-with a new party id in line with the new participant namespace. Therefore, we do not need to preserve the original
-root namespace key for a future migration. This works even if the domain nodes don't share the same protocol version.
-
-
+and namespace <participant_kms_migration_new_namespace>`. The first option
+does not require us to change the party ids for the active contracts in the old participant, and it can be done
+by a single operator, because it is a transparent process for the other operators and their shared contracts. However, both
+old and new domain have to use the same protocol version and the original root namespace key must be kept safe to allow
+the future migration to a new protocol version.
+The second option re-writes the contracts with new party identifiers in line with the new participant namespace.
+Therefore, we do not need to preserve the original root namespace key for a future migration. This works even if the
```

Comment:
```suggestion
Therefore, you do not need to preserve the original root namespace key for a future migration. This works even if the
```
According to the style guide, using "we" and "one" as subjects is discouraged. Instead, "you" should be used.